modules/virt-configuring-vm-use-usb-device.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt//virtual_machines/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
```

Comment:
Nothing I'd stop a merge review over, but if you're ever in these docs again and want to clean them up:
```suggestion
// * virt/virtual_machines/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
```